content/_index.de.html

```diff
@@ -12,85 +12,114 @@
 
 {{< blocks/cover title="OpenBikeSensor" image_anchor="top" height="400" >}}
 <div class="mx-auto">
-	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="https://github.com/openbikesensor">
-		Github <i class="fab fa-github ml-2 "></i>
+
+	<h1>Überholabstandsmessung für Radfahrende.</h1>
+	<p class="mb-5">Wir stehen für Open Innovation, Open Source, Open Data & Open Science.</p>
+
+	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="#introduction">
+		<i class="fas fa-arrow-down"></i> Mehr erfahren
 	</a>
-  <a class="btn btn-lg btn-secondary mr-3 mb-4" href="{{< relref "/slack" >}}">
-		Slack<i class="fab fa-slack ml-2 "></i>
+
+	<a class="btn btn-lg btn-secondary mr-3 mb-4" href="https://twitter.com/openbikesensor" target="_blank" rel="noopener noreferrer nofollow">
+		<i class="fab fa-twitter ml-2 "></i> @OpenBikeSensor
 	</a>
-	<a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/docs" >}}">
-		Dokumentation <i class="fas fa-arrow-alt-circle-right ml-2"></i>
```

Comment:
I think the Slack, Github and Documentation buttons are the more important, right in the center of the stage. The people come to the website mostly looking for the docs or source code (Github), and we want to guide them to our Slack primarily, not to Twitter.

Also, the "Mehr erfahren" button scrolls down, which is not required, as people naturally scroll down to see what's coming. So we can remove that button and reintroduce the other, important ones.

I like the "Unterstützen" (Support) button. 

I think 4 buttons is acceptable, for me that would be:

* Slack
* Github
* Dokumentation
* Unterstützen

Reply:
Nein. Diese Buttons kommen direkt darunter. Das wäre doppelt gemoppelt. Wir wollen ja, dass sie auch verstehen, wofür die Buttons gut sind. Daher der Bereich darunter. Was ich langfristig noch umsetzen möchte - da habe ich jedoch nicht gefunden, wie es geht - dass man auf die Icons klicken kann und zur entsprechenden Seite gelinkt wird.

Reply:
Ah now I see how you swapped around the references. Well, okay.

I still don't like buttons that scroll inline, but oh well. Let's keep them for now.